pom.xml

```diff
@@ -118,19 +118,49 @@
 	</scm>
 	<build>
 		<plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>buildnumber-maven-plugin</artifactId>
+                <version>1.4</version>
+                <executions>
+                    <execution>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>create</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <doCheck>false</doCheck>
```

Comment:
If we set this to `true` the build will fail on uncommited changes. I think it's better at `false` for contributors, but we can change it.